src/main/java/com/google/devtools/build/lib/remote/logging/proto/BUILD

```diff
@@ -0,0 +1,28 @@
+package(default_visibility = ["//src:__subpackages__"])
+
+load("//tools/build_rules:utilities.bzl", "java_library_srcs")
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//src/main/java/com/google/devtools/build/lib:__pkg__"],
+)
+
+java_library_srcs(
+    name = "dist_jars",
+    deps = [":rpc_log_entry_java_proto"],
+)
+
+java_proto_library(
+    name = "rpc_log_entry_java_proto",
+    deps = ["rpc_log_entry_proto"],
+)
+
+proto_library(
+    name = "rpc_log_entry_proto",
```

Comment:
I'm not sure this should live here, rather than src/main/protobuf. I see only one example of a proto file living outside of src/main/protobuf, need to ask Jakob about this. 
Also, there's a good chance this proto will live in build_event_stream anyway, if we decide to use BEP for actual logging, rather than our custom code.

Reply:
I moved it for now. I'll revert it if we need to go back